Makefile

```
@@ -23,9 +24,9 @@ run:
 release:
 	go mod download
 	$(MAKE) test
-	GOOS=linux GOARCH=amd64 go build -o build/${BINARY}-linux-amd64 ./cmd/dicomutil;
-	GOOS=darwin GOARCH=amd64 go build -o build/${BINARY}-darwin-amd64 ./cmd/dicomutil;
-	GOOS=windows GOARCH=amd64 go build -o build/${BINARY}-windows-amd64.exe ./cmd/dicomutil;
+	GOOS=linux GOARCH=amd64 go build -ldflags="-X 'main.Version=${VERSION}'" -o build/${BINARY}-linux-amd64 ./cmd/dicomutil;
+	GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'main.Version=${VERSION}'" -o build/${BINARY}-darwin-amd64 ./cmd/dicomutil;
+	GOOS=windows GOARCH=amd64 go build -ldflags="-X 'main.Version=${VERSION}'" -o build/${BINARY}-windows-amd64.exe ./cmd/dicomutil;
```

Does `main.Version` need to be changed to `main.GitVersion` here?

```suggestion
	GOOS=linux GOARCH=amd64 go build -ldflags="-X 'main.GitVersion=${VERSION}'" -o build/${BINARY}-linux-amd64 ./cmd/dicomutil;
	GOOS=darwin GOARCH=amd64 go build -ldflags="-X 'main.GitVersion=${VERSION}'" -o build/${BINARY}-darwin-amd64 ./cmd/dicomutil;
	GOOS=windows GOARCH=amd64 go build -ldflags="-X 'main.GitVersion=${VERSION}'" -o build/${BINARY}-windows-amd64.exe ./cmd/dicomutil;
```

---

> Does main.Version need to be changed to main.GitVersion here?

Yes it is. I've committed the suggestion.